public/static/Home.html

```diff
@@ -1,44 +1,50 @@
+<!DOCTYPE html>
+<html>
+
 <head>
 <title>marleigh - Home</title>
 </head>
 
 <body>
-<h1>HOME</h1>
-<br>
-<br>
-<br>
-<nav>
+  <div class = "whole-page">
+<h1 class = "main">HOME</h1>
   <nav>
   <a href="about_me">Me</a> |
-  <a href="current_projects">Current Projects </a> |
+  <a href="projects">Current Projects </a> |
   <a href="blog">Blog</a> |
-</nav>
-</nav>
+  <a href="espresso">Espresso</a> |
 
-<br>
-<br>
+  </nav>
 
-<img src= "images/charleston.jpg" alt="Charleston, South Carolina sunset " align="middle"> 
 
-<br>
-<br>
+<div class = "proper">
+<img src= "images/charleston.jpg" alt="Charleston, South Carolina sunset " align="middle"> 
+</div>
```

Comment:
Style-wise, this is a line you'd want to indent, as the `img` line is contained "inside" the `<div>`  Indentation and spacing helps establish a pattern for your eyes to quickly scan the code and see the structure of what's going on without actually having to read every line (might sound impossible now but trust me it'll happen!)

```html
<div class = "proper">
  <img src= "images/charleston.jpg" alt="Charleston, South Carolina sunset " align="middle"> 
</div>
```